source/trx2junit/Internal/JUnitBuilder.cs

```diff
@@ -59,36 +59,38 @@ private void AddTestSuite(string testSuiteName, IEnumerable<KeyValuePair<Guid, T
         //---------------------------------------------------------------------
         private void AddTest(XElement xTestSuite, KeyValuePair<Guid, TestDefinition> test)
         {
-            _testCount++;
+            var unitTestResults = _test.UnitTestResults[test.Key];
```

Comment:
```suggestion
            ICollection<UnitTestResult> unitTestResults = _test.UnitTestResults[test.Key];
```

But this will be changed either way.